.bcr/template.metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+  "homepage": "https://github.com/bazelbuild/rules_python",
+  "maintainers": [
+    {
+      "name": "Richard Levasseur",
+      "email": "rlevasseur@google.com",
+      "github": "rickeylev"
+    },
+    {
+      "name": "Greg Roodt",
+      "email": "groodt@gmail.com",
+      "github": "groodt"
+    }
+  ],
```

Comment:
This is a template for an entry that will appear the Bazel Central Registry ([example](https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/aspect_rules_js/metadata.json)).

I wasn't sure who the main maintainers of rules_python are. @rickeylev @groodt are you okay with being listed here?

Reply:
SGTM

Reply:
@kormide you can also add me.